packages/client/src/pages/admin/index.vue

```diff
@@ -35,292 +35,279 @@ import { instance } from '@/instance';
 import * as symbols from '@/symbols';
 import * as os from '@/os';
 import { lookupUser } from '@/scripts/lookup-user';
+import { MisskeyNavigator } from '@/scripts/navigate';
 
-export default defineComponent({
-	components: {
-		MkSuperMenu,
-		MkInfo,
-	},
+const isEmpty = (x: any) => x == null || x === '';
```

Comment:
I think this is a type error, what about this:
```suggestion
const isEmpty = (x: string | null) => x == null || x === '';
```
Since you use it for `maintainerName` and `maintainerEmail` only, that matches the [type definition in the Misskey SDK](https://github.com/misskey-dev/misskey.js/blob/c17c42fa05bcbcc38c7a4395d1bb3d2336d3d402/src/entities.ts#L256-L257)